src/components/pdf-viewer/Page.js

```diff
@@ -1,24 +1,32 @@
-import { forwardRef, useEffect, useState } from 'react'
+import { observer } from 'mobx-react-lite'
+import { forwardRef, useCallback, useEffect, useState } from 'react'
+
+import { useSharedStore } from '../SharedStoreProvider'
 
 import AnnotationLayer from './layers/AnnotationLayer'
 import CanvasLayer from './layers/CanvasLayer'
 import SVGLayer from './layers/SVGLayer'
 import TextLayer from './layers/TextLayer'
 
-export default forwardRef(function Page(
+// eslint-disable-next-line react/display-name
+const Page = observer(forwardRef((
```

Comment:
why not `export const Page`?

Reply:
Because the previous version exported default as well: `export default forwardRef`, and I assumed we want to keep it like this.